lib/run.js

```diff
@@ -5,6 +5,18 @@ const URL = require('url')
 const Histogram = require('native-hdr-histogram')
 const Client = require('./httpClient')
 const percentiles = require('./percentiles')
+const xtend = require('xtend')
+
+const defaultOptions = {
+  headers: {},
+  body: new Buffer(0),
+  method: 'GET',
+  duration: 10,
+  connections: 10,
+  pipelining: 1,
+  timeout: 10,
+  requests: [{}]
```

Comment:
I don't think it is, if the users just pass headers and a body I build it into a single request object in the array, which makes the requests collection usable for a single or multiple requests